### ISSUE TITLE
Command event-store:projection:names does not use provided limit

### ISSUE THREAD

**Issue by Miljar** [prooph/event-store-symfony-bundle#51]
When providing a limit, it is not used:
```shell
$php bin/console event-store:projection:names -l 100
```

Returns the default amount of 20 projection names.

When I look into the code, the provided limit is simply not passed to the method for fetching the projection names:

```php
            } else {
                $projectionNames = $projectionManager->$method($filter); // <-- $limit & $offset
            }
```

**Comment by prolic** [comment]
@Miljar Can you submit a PR and include some tests?

**Comment by codeliner** [comment]
Closing because bug solved with PR #52
